pkg/app/registry/micro/micro_test.go

```diff
@@ -31,6 +32,10 @@ import (
 	mreg "go-micro.dev/v4/registry"
 )
 
+func init() {
+	os.Setenv("MICRO_REGISTRY", "memory")
```

Comment:
A cleaner alternative would be to initialize the singleton directly in the init function, it's also what the guys did in the edge branch.

```suggestion
	_ = oreg.GetRegistry(oreg.Inmemory())
```